fork-report/Jenkinsfile

```
@@ -0,0 +1,39 @@
+@Library('pipeline-library@pull/348/head') _
+
+pipeline {
+	triggers {
+    // Execute once per hour
+		cron('@hourly')
+	}
+	options {
+		// This pipeline takes 1-2 minutes max to execute
+		timeout(time: 10, unit: 'MINUTES')
+    lock(resource: 'infra-reports-github-forks', inversePrecedence: true)
+	}
+	agent {
+    kubernetes {
+      yamlFile 'JenkinsAgentPodTemplate.yaml'
+    }
+  }
+	stages {
+    stage('Generate GitHub Forks Report') {
+      environment {
+        GITHUB_AUTH = credentials('jenkins-infra-reports')
+      }
+      steps {
+        dir('fork-report') {
+          // Requires $GITHUB_AUTH_PSW to hold an IAT (github installation access token) valid for 1 hour, or a PAT
```

```suggestion
          // Requires jenkins-infra-admin to be of type GithubAppCredentials so $GITHUB_AUTH_PSW holds an IAT (Github Installation Access Token) valid for 1 hour
```
I would mention the PAT usage only in the fork-report.rb script when wanting to launch this script locally.

---

then, better to not comment at all here (gotta pick your comment to move next to the credential)